stage1/stop/stop.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/shirou/gopsutil/process"
+)
+
+var (
+	force bool
+)
+
+func init() {
+	flag.BoolVar(&force, "force", false, "Forced stopping")
+}
+
+func readIntFromFile(path string) (i int32, err error) {
+	b, err := ioutil.ReadFile(path)
+	if err != nil {
+		return
+	}
+	_, err = fmt.Sscanf(string(b), "%d", &i)
+	return
+}
+
+func main() {
+	flag.Parse()
+
+	pid, err := readIntFromFile("ppid")
```

Comment:
This "stop" binary only reads "ppid" and not "pid". This is ok for now since this "stop" binary is only used for the coreos & fly flavors and not the kvm flavor and right now, only the kvm flavor uses "pid" instead of "ppid".

However, please note that https://github.com/coreos/rkt/pull/2799 changes this. So when both this PR and #2799 get merged, "rkt stop" on the fly flavor will need to be fixed here.

/cc @s-urbaniak